lerna.json

```diff
@@ -0,0 +1,8 @@
+{
+  "lerna": "2.11.0",
+  "packages": [
+    ".",
```

Comment:
I hope it's okay to have a package at the root. Either way, later we may wish to refactor by moving the core module under the `packages` directory.

Reply:
Me too. Definitely might find ourselves fighting Lerna on this one.